src/server/index.js

```diff
@@ -9,8 +9,14 @@ const http = require('http');
 const stream = require('stream');
 const fs = require('fs');
 const Promise = require('bluebird');
+// const cron = require('node-cron');
 
 const config = require('../config');
+const update = require('./update');
```

Comment:
I would prefer that you be more specific in this module name.  It sounds too generic to the point of confusion.

If the goal of this module is group all tasks that we may need now and in the future that update periodically into a single module, then it needs a proper name.

Reply:
Something like `updateExternalDataFiles` or `refreshExternalDataFiles`?

Reply:
I guess so.  It depends how you see this module changing over time.  Do you think envision using more cron jobs in the future?  If so, then this should be named to imply that it is related to cron specific jobs.

Reply:
Maybe just `update-cron`